www/main.css

```diff
@@ -56,8 +55,23 @@
   box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 }
 
-.use-me {
-  margin-left: 100px;
+.theme heaven-bound {
+  font-family: 'Pacifico',cursive;
+  text-shadow:
+  -1px -4px 0 #1D3170,
+  1px -4px 0 #1D3170,
+  -1px 4px 0 #1D3170,
+  1px 4px 0 #1D3170;
+  position: relative;
+  color: #fff;
+}
+.theme heaven-bound span {
+  background: url(landing/www/gradient-dark.png) repeat-x;
```

Comment:
Don't use any images for gradients. Just try to use CSS like so: http://css-tricks.com/snippets/css/gradient-text/.

It's OK if we don't support every browser. Just let it have a decent default color if the gradient style is unsupported.